src/views/projects/Header.svelte

```diff
@@ -19,6 +20,8 @@
 
   export let openPatchCount: number;
   export let openIssueCount: number;
+
```

Comment:
Can you remove this line? So the props are in one block.

Reply:
[diff](https://github.com/radicle-dev/radicle-interface/compare/66d3e1496e2f5179e58be4af261c99494752b2d0..c3346575eece8c059370a629cea73470ae16ce78)